docs/content/preview/yugabyte-voyager/migrate/live-fall-back.md

```diff
@@ -649,6 +650,12 @@ The export directory has the following sub-directories and files:
 - `metainfo` and `temp` directories are used by yb-voyager for internal bookkeeping.
 - `logs` directory contains the log files for each command.
 
+## Assess Migration
+
+This step is optional and only required for PostgreSQL and Orcale Database Migration. Assess migration assess the migration complexity and get schema changes, data distribution and cluster sizing recommendations using the `yb-voyager assess` command.
```

Comment:
```suggestion
This step is optional and only applicable to PostgreSQL and Oracle database migrations. Assess migration analyzes the source database, captures essential metadata, and generates a report with recommended migration strategies and cluster configurations for optimal performance with YugabyteDB. You run assessments using the `yb-voyager assess` command.
```

Reply:
the command is `yb-voyager assess-migration`